crates/aptos-inspection-service/src/server/mod.rs

```diff
@@ -132,6 +143,41 @@ async fn serve_requests(
             // Exposes the system and build information
             system_information::handle_system_information_request(node_config)
         },
+        CPU_PROFILING_PATH => {
```

Comment:
Ideally, we should be able to start a separate profiling service independent of inspection service with its own dedicated URL mapping.